apps/src/sites/studio/pages/levels/editors/_applab.js

```diff
@@ -11,21 +11,23 @@ import {
 } from '@cdo/apps/lib/kits/maker/dropletConfig';
 
 $(document).ready(function() {
-  // Get the set of blocks for the Maker Category, the Circuit Category, and the Micro:bit category
+  $('#level_makerlab_enabled').change(function() {
+    // Get the set of blocks for the Maker Category, the Circuit Category, and the Micro:bit category
 
-  // The maker type given here sets the defaultPin in the example block. Since we are just using the function name,
-  // which doesn't include a pin parameter, we could any block type.
-  let makerBlocks = getMakerBlocks(null);
-  // Setting block values to null to match the expected behavior in code_functions.
-  makerBlocks = makerBlocks.forEach(block => (makerBlocks[block.func] = null));
-  const microbitBlocks = {};
-  configMicrobit.blocks.forEach(block => (microbitBlocks[block.func] = null));
-  const circuitBlocks = {};
-  configCircuitPlayground.blocks.forEach(
-    block => (circuitBlocks[block.func] = null)
-  );
+    // The maker type given here sets the defaultPin in the example block. Since we are just using the function name,
+    // which doesn't include a pin parameter, we could use any block type.
+    const configMaker = getMakerBlocks(null);
+    let makerBlocks = {};
+    // Setting block values to null to match the expected behavior in code_functions.
```

Comment:
Nit: move this comment above the previous line, and remove blank line between configMaker.forEach and let microbitBlocks to make clear this comment applies to all of these 6 lines (if I understand it correctly?)